app/src/tft.cpp

```diff
@@ -220,47 +220,45 @@ void tftUpdateDisplay()
   }
 }
 
+const char *hvac_mode_str[NR_MODES] = {
+  "off", "auto", "heating", "cooling", "fan-only", "aux-heat", "error", "idle",
```

Comment:
After studying the code more in-depth, I now recall why things seem to be duplicated. Home Assistant/MQTT requires the lower case strings and ending with "ing" listed in hvacModeToMqttCurrMode(). The string returned from hvacModeToString() is specifically for the user interface. They look nicer on the screen. How about adding another parameter to indicate what string the caller is looking for?

Reply:
Steve, 

2 quick clarification question:

a) is this the MQTT/Home assistant info I should follow for these string modes?; and
b) do you want the mode words with their first character capitalized for the UI?

Cheers!

Reply:
> a) is this the MQTT/Home assistant info [1] I should follow for these string modes?
Yes!!
> b) do you want the mode words with their first character capitalized for the UI?
Yes!!
> 
> [1] https://www.home-assistant.io/integrations/climate.mqtt/